neo/Consensus/ConsensusService.cs

```diff
@@ -46,6 +46,7 @@ public ConsensusService(IActorRef localNode, IActorRef taskManager, Store store,
             this.taskManager = taskManager;
             this.store = store;
             this.context = context;
+            Context.System.EventStream.Subscribe(Self, typeof(Blockchain.PersistCompleted));
```

Comment:
Perhaps I'll need to update Unit tests for this Erik... I remember that TesKit also had an EventStream system. I'll study more, let's see.